### PR TITLE
Update input workbook: v20220902

### PR DESCRIPTION
**Pull request by ajhyndman**
Update the committed input template to the latest version and rebuild validation rules

### CODE REVIEW

File: scripts/generateRules.mjs [hunk 1]
```diff
@@ -290,27 +290,6 @@ function validateExtractedDropdowns(extractedDropdowns) {
...
-  // Make sure that every correction configured refers to a real value in a dropdown somewhere
-  Object.keys(dropdownCorrections).forEach(valToCorrect => {
-    for (const dropdownName in extractedDropdowns) {
-      for (const dropdownValue of extractedDropdowns[dropdownName]) {
-        if (valToCorrect == dropdownValue) {
-          return
-        }
-      }
-    }
-    // Didn't find the value anywhere? Throw an error to force us to correct it
```
Comment (ajhyndman) [inline scripts/generateRules.mjs:303]: @trumanc I think this validation would prevent us from keeping corrections for uploads from old template versions, is that not correct?

I had to remove it to regenerate rules with the latest workbook, but let me know if I'm missing something.

Comment (trumanc) [reply scripts/generateRules.mjs:303]: I wrote this in a way that will allow it to be compatible with old workbooks.
The idea is that you'll need to update the configured corrections here: https://github.com/usdigitalresponse/arpa-reporter/blob/staging/src/server/services/validation-rules.js#L24

They key of that map needs to be updated to the new correct value that is in in the workbook, and the old value needs to be added to the `allowableLegacyValues` list (creating a new list if it doesn't exist yet.

Comment (ajhyndman) [reply scripts/generateRules.mjs:303]: Whoops, thanks!  Now that I read your response, I feel like you did explain it to me once already.  My bad.

I'll make another pass, updating the correction configurations.

**Review by trumanc — CHANGES_REQUESTED** [review]
I spent a bunch of time testing uploads in the Treasury portal and determined the validity of a bunch of these different dropdown values. Here is what I found:

BAD VALUES:
Childcare, daycare, and early learning facilities
COVID-19 testing sites and laboratories and acquisition of related equipment
COVID-19 testing sites and laboratories, and acquisition of related equipment
Mitigation measures in small businesses, nonprofits, and impacted industries
Affordable housing supportive housing or recovery housing


GOOD VALUES:
Childcare, daycare and early learning facilities
COVID-19 testing sites and laboratories
Mitigation measures in small businesses, nonprofits and impacted industries
Affordable housing, supportive housing, or recovery housing


With that in mind, the new `dropdownCorrections` object in the validation-rules.js file should look like this:
```
const dropdownCorrections = {
  'Affordable housing supportive housing or recovery housing': {
    correctedValue: 'Affordable housing, supportive housing, or recovery housing'
  },
  'Childcare, daycare, and early learning facilities': {
    correctedValue: 'Childcare, daycare and early learning facilities'
  },
  'COVID-19 testing sites and laboratories': {
    allowableLegacyValues: [
      'COVID-19 testing sites and laboratories, and acquisition of related equipment',
      'COVID-19 testing sites and laboratories and acquisition of related equipment'
    ]
  },
  'Mitigation measures in small businesses, nonprofits, and impacted industries': {
    correctedValue: 'Mitigation measures in small businesses, nonprofits and impacted industries'
  },
  'Family or child care': {
    allowableLegacyValues: ['Family or childcare']
  }
}
```

And then you'll be able to add back in the validations in the generateRules script

**Pull request closed without merging**